### PR TITLE
fix: pagination/searching/sorting in billing invoice table

### LINKED ISSUE #3472: pagination/searching/sorting doesn't work in billing invoice table
**Describe the bug**

The pagination/searching/sorting in event invoice table in billing info isn't working

**Expected behaviour**

It should work fine

**Additional context**

On it

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #3472 

#### Short description of what this resolves:
The pagination/searching/sorting in event invoice table in billing info isn't working

#### Changes proposed in this pull request:
- change searching to `status` attribute as there is no other searchable attribute
- add sort options to status and amount
- fix pagination by passing correct data in the list template

eventInvoice-2019-08-30_221605

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/routes/account/billing/invoices/list.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ export default class extends Route.extend(EmberTableRouteMixin) {
 
     queryString = this.applySortFilters(queryString, params);
     return {
-      eventInvoices: (await this.store.query('event-invoice', queryString)).toArray(),
+      eventInvoices: await this.asArray(this.store.query('event-invoice', queryString)),
```
Comment (kushthedude) [inline app/routes/account/billing/invoices/list.js]: Remove the complete return block and use
```js
return  this.asArray(this.store.query('event-invoice', queryString));
```

Comment (shreyanshdwivedi) [reply app/routes/account/billing/invoices/list.js]: The params being passed here are used in the controller. @kushthedude

Comment (mrsaicharan1) [inline app/routes/account/billing/invoices/list.js]: Are the event invoices being returned from the model properly here?

Comment (shreyanshdwivedi) [reply app/routes/account/billing/invoices/list.js]: Yes

**Comment by shreyanshdwivedi** [comment]
@kushthedude @uds5501 @mrsaicharan actually I'm traveling, so can you guys please pull the changes and review on your local. It'll be really helpful.
Thanks

**Comment by shreyanshdwivedi** [comment]
@uds5501 @mrsaicharan1 @kushthedude I've added a gif for reference in the PR description. Please have a look

**Pull request merged**